python-sdk/langwatch/examples/migration_support/langchain_rag_bot.py

```
@@ -0,0 +1,104 @@
+from langchain.schema import HumanMessage
+from langchain.schema.runnable.config import RunnableConfig
+
+import langwatch
+
+from langchain_community.document_loaders import WebBaseLoader
+from langchain_community.vectorstores.faiss import FAISS
+from langchain_openai import ChatOpenAI, OpenAIEmbeddings
+from langchain_text_splitters import RecursiveCharacterTextSplitter
+from langchain.tools.retriever import create_retriever_tool
+from langchain.agents import AgentExecutor, create_tool_calling_agent
+from langchain.tools import BaseTool, StructuredTool, tool
```

Ruff (0.8.2)

`langchain.tools.BaseTool` imported but unused

Remove unused import

(F401)